sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.2.0-beta.2 (Unreleased)
 
 ### Features Added
+- Added property `RecognizeCustomEntitiesAction` to `TextAnalyticsActions` to support the new 'Recognize Custom Entities' API. This action can be used to get custom entities for an input document or batch of documents.
```

Comment:
This is what I wrote in JS's changelog which I adapted to use .NET names:
```suggestion
- We are now targeting the service's v3.2-preview.2 API as the default instead of v3.2-preview.1.
- Adding support for a three new actions in `StartAnalyzeActions`: `RecognizeCustomEntities`, `SingleCategoryClassify`, and `MultiCategoriesClassify`. The new actions allow you to use custom models to perform entity recognition and classification actions.
```

Reply:
This looks good!